### PR TITLE
Added Web-ifc-viewer hello-world

### PR DESCRIPTION
**Pull request by AnweshGangula**
Added documentation to creating a hello-world project for web-ifc-viewer (IFCjs/info#25)

### CODE REVIEW

File: docs/Guide/web-ifc-viewer/Introduction.mdx [hunk 1]
```diff
@@ -3,10 +3,124 @@ title: Introduction
...
 
-## 🚧👷‍♀️ Under construction 👷‍♂️🚧
+## Introduction
 
-<IfcCard icon="💙" title="Thanks for your interest in IFC.js!">
-    We are building this section of the documentation right now. In the meantime, you can join the <a href="https://discord.gg/FXfyR4XrKT">Discord channel</a> and ask us any questions you need to implement IFC.js or to join the project.
-</IfcCard>
\ No newline at end of file
+As discussed in the [Getting Started](../Getting%20started.mdx) guide - web-ifc-three viewer is a 3D BIM viewer with many tools and functionalities already implemented (section drawings, dimensions, etc.), allowing you to create BIM tools with very little effort.\
+It is suggseted to use this when you want to create a BIM viewer and you don't want to spend time implementing all the model navigation tools you would like to have.
```
Comment (agviegas) [inline docs/Guide/web-ifc-viewer/Introduction.mdx:13]: suggested*

**Comment by agviegas** [comment]
Hey @AnweshGangula the content itself is perfect, but the tone is a bit neutral when compared with other parts of the docs :) If you could try to make the style more informal and add emojis like in other parts of the documentation, it would be perfect. Thanks!

**Comment by AnweshGangula** [comment]
@agviegas, check out the changes and let me know your thoughts

**Comment by agviegas** [comment]
Sorry, didn't have time to check out this earlier @AnweshGangula. Looks great for me, so this closes this bounty! 🙂 To get the money of the bounty, you can create an OpenCollective account (if you don't have one yet), create a super simple invoice saying that you did this bounty and submit it [here](https://opencollective.com/ifcjs). I'll approve it right away and you'll get it. If you have any questions, feel free to reach out in Discord!

**Pull request merged**